src/views/Project/index.jsx

```diff
@@ -79,22 +152,44 @@ export default class Project extends Component {
           'state:open',
         ].join(' ');
 
-        return this.fetch(searchQuery);
+        return this.fetchGithub(searchQuery);
+      })
+    );
+    const variable = {
+      search: {
+        tags: ['good-first-bug'],
+        statuses: ['NEW', 'UNCONFIRMED', 'ASSIGNED', 'REOPENED'],
+      },
+      paging: {
+        page: 0,
+        pageSize: 100,
+      },
+    };
+    const productWithComponentList = mergeAll(
+      project.products.filter(product => typeof product !== 'string') || []
```

Comment:
Running the app locally and opening and opening `http://localhost:5000/projects/servo` I get `project.products is undefined` in the console with an infinite spinner.

Reply:
ok fixed